### PR TITLE
Tweak 3x8_docs

### PR DESCRIPTION
**Pull request by joao-paulo-parity**
From my review at https://github.com/paritytech/pr-custom-review/pull/65#pullrequestreview-899926775

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +31,7 @@ Upon receiving [pull_request](https://docs.github.com/en/actions/learn-github-ac
 - `diff` which matches a rule based on the PR's diff content
 - `changed_files` which matches a rule based on paths/files changed in the PR
 
-If a given rule is matched and its approval count is not met, then reviews will be requested from the missing users/teams for that rule and a failed commit status will be set for the PR; this status can be made a requirement through branch protection rules in order to block the PR from being merged until all conditions are passing (see [GitHub repository configuration](#github-repository-configuration)).
+If a given rule is matched and its approval count is not met, then reviews will be requested from the missing users/teams for that rule and a failed commit status will be set for the PR; this status can be made required through branch protection rules in order to block the PR from being merged until all conditions are passing (see [GitHub repository configuration](#github-repository-configuration)).
```
Comment (TriplEight) [inline README.md:34]:
```suggestion
If a given rule is matched and its approval count is not met, then reviews will be requested from the missing users/teams for that rule and a failed commit status will be set for the PR; this status should be made required through branch protection rules in order to block the PR from being merged until all conditions are passing (see [GitHub repository configuration](#github-repository-configuration)).
```

File: README.md [hunk 2]
```diff
@@ -292,20 +300,42 @@ installed.
...
+
+    As of this writing, the secret setup can be done in
+    `https://github.com/organizations/${ORG}/settings/secrets/actions`.
+    For further context see
+    [Creating encrypted secrets for an organization](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-an-organization)
+    and
+    [Using encrypted secrets in a workflow](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow).
+
+4. Add the [configuration file](#configuration-file) to the repository's
+  default branch, e.g. https://github.com/paritytech/substrate/pull/10968/files.
```
Comment (TriplEight) [inline README.md:326]:
```suggestion
  default branch, e.g. <https://github.com/paritytech/substrate/pull/10968/files>.
```
no-bare-urls lint

File: README.md [hunk 3]
```diff
@@ -203,8 +209,10 @@ jobs:
       - name: pr-custom-review
         uses: paritytech/pr-custom-review@tag           # Pick a release tag and put it after the "@".
         with:
-          # A token with read-only organization permission is required for
-          # requesting reviews from teams.
+          # The token needs the following permissions
+          # - `read:org` for being able to request reviews from teams
+          # - `workflow` for being able to request the workflow's job
+          #    information; used to track lines in the job's output
```
Comment (TriplEight) [inline README.md:215]: Don't you think this (or at least the required scopes) should be mentioned in the p2 of the Deployment chapter?

File: README.md [hunk 4]
```diff
@@ -292,20 +300,42 @@ installed.
...
+    and
+    [Using encrypted secrets in a workflow](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow).
+
+4. Add the [configuration file](#configuration-file) to the repository's
+  default branch, e.g. https://github.com/paritytech/substrate/pull/10968/files.
+  This should be done in a separate PR prior to adding the workflow
+  configuration in the next step.
+
+5. Add the [workflow configuration](#workflow-configuration), e.g.
+  https://github.com/paritytech/substrate/pull/10951/files
```
Comment (TriplEight) [inline README.md:331]:
```suggestion
  <https://github.com/paritytech/substrate/pull/10951/files>
```
no-bare-urls lint

**Review by TriplEight — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (joao-paulo-parity) [reply README.md:215]: 8ae78bede894f3a5400398746af41871fa8e57ff

**Review by TriplEight — APPROVED** [review]

**Review by TriplEight — APPROVED** [review]

**Pull request merged**